kanbantool_ticket_dash/utils.py

```diff
@@ -22,7 +22,9 @@ def fetch_tickets():
     tickets = json.loads(r.text)
 
     for ticket in tickets:
-        ticket['task']['created_at'] = arrow.get(ticket['task']['created_at'])
+        # Creating task key, because the rest of the app depends on it.
```

Comment:
Just add another comment line stating that this is to deal with an API change on kanbantool's side